docs/source/reference/router/telemetry/instrumentation/standard-instruments.mdx

```diff
@@ -117,6 +117,11 @@ The initial call to Uplink during router startup is not reflected in metrics.
   - `report.type`: The type of report submitted: "traces" or "metrics"
   - `report.protocol`: Either "apollo" or "otlp", depending on the experimental_otlp_tracing_sampler configuration.
 
+### Telemetry
+- `apollo.router.telemetry.batch_processor.errors` - The number of errors encountered by exporter batch processors.
+  - `name`: One of `apollo-tracing`, `datadog-tracing`, `otlp-tracing`, `zipkin-tracing`.
```

Comment:
SHould it also include `jaeger-collector` ?

Reply:
Yes, Fixed in [30ed671](https://github.com/apollographql/router/pull/6558/commits/30ed6715416e91bd07bc0cbebbe0b3a8aae229fe)